packages/fast-components-styles-msft/src/toggle/index.ts

```diff
@@ -137,45 +145,60 @@ const styles: ComponentStyles<ToggleClassNameContract, DesignSystem> = {
                     neutralFocusInnerAccent(accentFillRest)
                 ),
                 borderColor: neutralFocus,
+                ...highContrastDoubleFocus,
             }),
-            "@media (-ms-high-contrast:active)": {
+            "&:disabled": {
+                [highContrastSelector]: {
+                    background: "GrayText !important",
+                    borderColor: "GrayText !important",
+                    "& + span": {
+                        background: "Background !important",
+                    },
+                },
+            },
+            [highContrastSelector]: {
                 background: "Highlight",
                 borderColor: "Highlight",
+                "&:hover": {
+                    background: "HighlightText",
+                    borderColor: "Highlight",
+                },
                 "&:active": {
-                    "@media (-ms-high-contrast:active)": {
+                    [highContrastSelector]: {
                         background: "Highlight",
                         "& + span": {
-                            background: "Background",
+                            background: "HighlightText",
                         },
                     },
                 },
-                "&:hover": {
-                    "@media (-ms-high-contrast:active)": {
-                        background: "Background",
-                        borderColor: "Highlight",
-                    },
-                },
             },
         },
         "& $toggle_stateIndicator": {
             left: toPx(indicatorCheckedLeft),
             background: accentForegroundCut,
-            "@media (-ms-high-contrast:active)": {
-                background: "Background",
+            [highContrastSelector]: {
+                background: "HighlightText",
+                "&:hover": {
+                    [highContrastSelector]: {

```

Comment:
are you sure this nesting is right?i wouldn't think you could put a hover on the high contrast selector, and even if you could you wouldn't then need to re-declare the selector

Reply:
weird, not sure why this didn't complain. updated to remove hover out of the high contrast selector